sa/rate_limits_test.go

```diff
@@ -80,9 +79,7 @@ func TestCertsPerNameRateLimitTable(t *testing.T) {
 		t.Run(tc.caseName, func(t *testing.T) {
 			timeRange := &sapb.Range{
 				EarliestNS: aprilFirst.Add(-1 * time.Second).UnixNano(),
-				Earliest:   timestamppb.New(aprilFirst.Add(-1 * time.Second)),
```

Comment:
These should actually still exist. They were introduced in https://github.com/letsencrypt/boulder/pull/7070. Can you merge main instead please?

Reply:
Oh gosh, yeah, this is actually a bad merge that was hidden by some some rebasing.

Reply:
All fixed, thanks for catching this!